incubator/binding-risingwave.spec/src/main/scripts/io/aklivity/zilla/specs/binding/risingwave/streams/transformation/create.materialized.view/client.rpt

```diff
@@ -0,0 +1,185 @@
+                              .typeId(zilla:id("pgsql"))
+                              .ready()
+                                .status("IDLE")
+                                .build()
+                              .build()}
+
+write zilla:data.ext ${pgsql:dataEx()
+                              .typeId(zilla:id("pgsql"))
+                              .query()
+                                .build()
+                              .build()}
+write "DESCRIBE MATERIALIZED VIEW active_users;"
+      [0x00]
+
+read advised zilla:flush ${pgsql:flushEx()
+                              .typeId(zilla:id("pgsql"))
+                              .type()
+                                .column()
+                                    .name("balance")
+                                    .tableOid(0)
+                                    .index(0)
+                                    .typeOid(701)
+                                    .length(8)
+                                    .modifier(-1)
+                                    .format("TEXT")
+                                    .build()
+                                .column()
+                                    .name("status")
+                                    .tableOid(0)
+                                    .index(0)
+                                    .typeOid(20)
+                                    .length(8)
+                                    .modifier(-1)
+                                    .format("TEXT")
+                                    .build()
+                                .column()
+                                    .name("user_id")
+                                    .tableOid(0)
+                                    .index(0)
+                                    .typeOid(17)
+                                    .length(-1)
+                                    .modifier(-1)
+                                    .format("TEXT")
+                                    .build()
+                                .build()
+                              .build()}
+
+read advised zilla:flush ${pgsql:flushEx()
+                              .typeId(zilla:id("pgsql"))
+                              .completion()
+                                .tag("DESCRIBE")
+                                .build()
+                              .build()}
+
+read advised zilla:flush ${pgsql:flushEx()
+                              .typeId(zilla:id("pgsql"))
+                              .ready()
+                                .status("IDLE")
+                                .build()
+                              .build()}
+
+
+
+write zilla:data.ext ${pgsql:dataEx()
+                              .typeId(zilla:id("pgsql"))
+                              .query()
+                                .build()
+                              .build()}
+write "CREATE SINK active_users_sink"
+      "FROM active_users"
+      "WITH ("
+      "   connector='kafka',"
+      "   topic='users',"
```

Comment:
Let's use `weather` and/or `cities` tables / topics throughout the scripts, instead of sometimes using `balances` so that it is generic to postgress, and not specific to streampay example.

See https://www.postgresql.org/docs/current/tutorial-table.html for table column names and types.